### PR TITLE
Several moth wing related fixes

### PR DESCRIPTION
**Pull request by CydiaLamiales**
## About The Pull Request
[#8062 ](https://github.com/BeeStation/BeeStation-Hornet/pull/8062) missed some stuff, namely clockwork wings, neck fluff for open wings, some antennae in the character creator and some code for the new markings as a separate mutant body part. This PR fixes that. Also it fixes the 'blacklist' of wings, body markings and antennae.

## Why It's Good For The Game
Consistency good, disappearing sprites on characters bad.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Clockwork wings are back (Obtained through Biometallic Replication):

clockwork_wings

Wing selection menu in character customization :

moth_wings_menu

Antennae selection menu in character customization :

moth_antennae_menu

Body Marking selection menu in character customization :

moth_markings_menu

Neck fluff showing while the wings are open : 

image

image

Solved by adding the fluff to the open wings file :

image

</details>

## Changelog
:cl:
del: Removed the Angel, Burnt Off and Clockwork wings, antennae and markings from the character customization menu
fix: Fixes moth neck fluff not showing when their wings are open
fix: Fixed wing, antennae and body marking blacklist not working
imageadd: Added back the sprite for clockwork wings
imageadd: Added added the neck fluff to the open wings file
code: Moth character customization and random mob spawning now use moth_wing_roundstart_list, moth_antennae_roundstart_list and moth_markings_roundstart_list for their respective body parts
/:cl:

### CODE REVIEW

**Comment by CydiaLamiales** [comment]
Guh, what's with all the labels.

File: code/_globalvars/lists/flavor_misc.dm [hunk 1]
```diff
@@ -38,8 +38,11 @@ GLOBAL_LIST_EMPTY(wings_list)
 GLOBAL_LIST_EMPTY(wings_open_list)
 GLOBAL_LIST_EMPTY(r_wings_list)
 GLOBAL_LIST_EMPTY(moth_wings_list)
+GLOBAL_LIST_EMPTY(moth_wings_sanitized_list)//this lacks the blacklisted wings such as burned, clockwork and angel
```
Comment (itsmeow) [inline code/_globalvars/lists/flavor_misc.dm]: name these roundstart_choices_list or something, sanitized doesn't really make sense

**Review by itsmeow — CHANGES_REQUESTED** [review]

**Comment by itsmeow** [comment]
> Guh, what's with all the labels. Administration, really? Or is it because I changed this? image

yes

File: code/_globalvars/lists/flavor_misc.dm (hunk 1 above)
Comment (CydiaLamiales) [reply code/_globalvars/lists/flavor_misc.dm]: That's the name I went with in the cocoon PR but then I realized this also goes into random mob creation and admin mob creation. But if you say so, I'll change it to roundstart_list

Comment (CydiaLamiales) [reply code/_globalvars/lists/flavor_misc.dm]: Done

**Comment by CydiaLamiales** [comment]
The bot really wants those labels to stick.

**Review by itsmeow — APPROVED** [review]

**Pull request merged**